content/articles/weebly-service.md

```diff
@@ -14,22 +14,26 @@ categories:
 
 ---
 
-[Weebly](http://www.weebly.com) gives millions of people a surprisingly easy and affordable way to create a site that is as unique as they are. With DNSimple's one-click [DNS services](/categories/services/), you can set up the DNS records required by Weebly.
+[Weebly](http://www.weebly.com) gives millions of people a surprisingly easy and affordable way to create a site that is as unique as they are. 
```

Comment:
I looked at linking their support doc for using a custom domain, but they immediately suggest transferring it to them, so I don't love that idea. They also include a lot of extra steps that we don't require with this being a one-click